gufe/tests/test_smallmoleculecomponent.py

```diff
@@ -251,6 +251,17 @@ def test_bounce_off_file(self, toluene, tmpdir):
 
         assert roundtrip == toluene
 
+    @pytest.mark.skipif(not HAS_OFFTK, reason="no openff toolkit available")
+    def test_to_openff_after_serialisation(self, toluene):
+        TOKENIZABLE_REGISTRY.pop(toluene.key)
```

Comment:
@dwhswenson is this still verboten and what's the correct thing to mock to not do this? (it is required for the test to correctly fail without the patch)

Reply:
I think what you need is something like this:

https://github.com/OpenFreeEnergy/gufe/blob/2a58ca07f81c77dbf3ad7beef643bee1f359d420/gufe/tests/test_tokenization.py#L109-L111

That acts as if nothing had been registered (fresh Python session).

If you need most of the registry, but want to remove one item, you can do it with something like what we do for `_set_key`:

https://github.com/OpenFreeEnergy/gufe/blob/2a58ca07f81c77dbf3ad7beef643bee1f359d420/gufe/tests/test_tokenization.py#L202-L207

Any changes inside the patch context will not affect the real dict, only the patched copy.